pages/resources.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: default
+permalink: /resources
+title: Student Resources
+---
+
+Next House has many resources for students to access from the Residential Peer Mentors Program to our Area Director and Heads of House. All the members of the community are there to help and support students and specifically freshmen as they accustom to Next and the Institvte.
+
```

Comment:
Yeah, I agree -- I'll add something to the css that puts a margin below each paragraph or something